src/server/routes/api/document/pubmed/searchPubmed.js

```diff
@@ -51,6 +54,86 @@ const eSearchPubmed = ( term, opts ) => {
   .then( pubmedDataConverter );
 };
 
+/**
+ * Search for an article using Crossref and return a Pubmed-style result.
+ * 
+ * This approach uses Crossref to find the article's DOI.  The DOI is sent to PMC to find a
+ * PMID.  If the PMC query fails for any reason, a query to Pubmed Eutils is made using the 
+ * DOI as the search term.  For some articles, PMC fails.  For others, Eutils fails.  Using
+ * both gives good coverage.
+ * 
+ * @param {String} term The search term to find the article (e.g. title)
+ * @param {Object} opts Pubmed API options
+ * @returns {Object} A Pubmed search result object
+ */
+const crSearchPubmed = async ( term, opts ) => {
```

Comment:
This should look at `opts` to see if the term looks like a 'title', 'doi' or 'pmid'. 

If its a pmid or doi, go direct to `eSearchPubmed` because the `crSearchForDoi` is hardcoded to fields`query.bibliographic`.